src/clients/kdump_auto.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
ah, I would not not add that automatically to files as it is potentially dangerous ( ruby will raise exception if code manipulate with frozen string literal and sadly old ycp code doing it quite lot.

Reply:
I have removed it